pkg/parse/parse.go

```diff
@@ -0,0 +1 @@
+// Helper package for processing bugzilla and trello data
```

Comment:
what does this package do?  Trello-specific logic should live in the trello package, bz-specific logic should live in the BZ package.  If you have common code that unifies the two, then it can live here, but it should be more descriptively named, with a better comment.